examples/z_bytes.c

```diff
@@ -200,9 +200,9 @@ int main(void) {
     {
         /// fill z_bytes with some data
         z_owned_bytes_t b1, b2, b3;
-        z_bytes_copy_from_str(&b1, "abc");
-        z_bytes_copy_from_str(&b2, "def");
-        z_bytes_copy_from_str(&b3, "hij");
+        z_bytes_from_str(&b1, "abc", NULL, NULL);
```

Comment:
using z_bytes_from_str/buf with NULL destructors, is not really encouraged, since zenoh does not provide the guarantee on how it will keep the pointers to data (especially in case of session-local communications). It is safe to use this functions exactly in this context since the strings are in static memory, but exactly for this  we have z_bytes_from_static_str/buff which do not require specifying destructors.

Reply:
Thanks. I was unaware of the existence of `z_bytes_from_static_str`. Would it be good to change this to `z_bytes_from_static_str`? This function has already been used in other examples.